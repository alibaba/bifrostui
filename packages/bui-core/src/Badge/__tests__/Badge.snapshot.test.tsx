import { snapshotTest } from 'testing';

describe('Badge snapshot', () => {
  snapshotTest('Badge');
});
