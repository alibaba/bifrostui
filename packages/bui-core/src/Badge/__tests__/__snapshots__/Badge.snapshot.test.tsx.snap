// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge snapshot Badge demo snapshot 0 1`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "row",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "20px",
    }
  }
>
  <div
    className="bui-badge bui-badge-position"
  >
    <div
      className="bui-badge-primary bui-badge-contained bui-badge-standard"
    >
      12
    </div>
    <div
      className="bui-avatar bui-avatar-medium bui-avatar-rounded"
    >
      <div
        className="bui-image bui-avatar-img"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      >
        <img
          className="bui-image-img bui-image-loaded"
          loading="eager"
          onError={[Function]}
          onLoad={[Function]}
          src="//perico.damai.cn/userheadphotos/927551/185510204.jpg"
          style={
            Object {
              "objectFit": "cover",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`Badge snapshot Badge demo snapshot 0 2`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
    }
  }
>
  <div
    className="bui-badge"
  >
    <div
      className="bui-badge-primary bui-badge-contained bui-badge-standard"
    >
      1
    </div>
  </div>
</div>
`;

exports[`Badge snapshot Badge demo snapshot 0 3`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "row",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "10px",
    }
  }
>
  <div
    className="bui-badge bui-badge-position"
  >
    <div
      className="bui-badge-primary bui-badge-contained bui-badge-dot"
    />
    <div
      className="bui-avatar bui-avatar-medium bui-avatar-rounded"
    >
      <div
        className="bui-image bui-avatar-img"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      >
        <img
          className="bui-image-img bui-image-loaded"
          loading="eager"
          onError={[Function]}
          onLoad={[Function]}
          src="//perico.damai.cn/userheadphotos/927551/185510204.jpg"
          style={
            Object {
              "objectFit": "cover",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`Badge snapshot Badge demo snapshot 0 4`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "row",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "10px",
    }
  }
>
  <div
    className="bui-badge bui-badge-position"
  >
    <div
      className="bui-badge-primary bui-badge-contained bui-badge-bubble"
    >
      12
    </div>
    <div
      className="bui-avatar bui-avatar-medium bui-avatar-rounded"
    >
      <div
        className="bui-image bui-avatar-img"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      >
        <img
          className="bui-image-img bui-image-loaded"
          loading="eager"
          onError={[Function]}
          onLoad={[Function]}
          src="//perico.damai.cn/userheadphotos/927551/185510204.jpg"
          style={
            Object {
              "objectFit": "cover",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`Badge snapshot Badge demo snapshot 0 5`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "row",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "10px",
    }
  }
>
  <div
    className="bui-badge bui-badge-position"
  >
    <div
      className="bui-badge-primary bui-badge-contained bui-badge-standard"
    >
      12
    </div>
    <div
      className="bui-avatar bui-avatar-medium bui-avatar-rounded"
    >
      <div
        className="bui-image bui-avatar-img"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      >
        <img
          className="bui-image-img bui-image-loaded"
          loading="eager"
          onError={[Function]}
          onLoad={[Function]}
          src="//perico.damai.cn/userheadphotos/927551/185510204.jpg"
          style={
            Object {
              "objectFit": "cover",
            }
          }
        />
      </div>
    </div>
  </div>
  <div
    className="bui-badge bui-badge-position"
  >
    <div
      className="bui-badge-primary bui-badge-contained bui-badge-bubble"
    >
      12
    </div>
    <div
      className="bui-avatar bui-avatar-medium bui-avatar-rounded"
    >
      <div
        className="bui-image bui-avatar-img"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      >
        <img
          className="bui-image-img bui-image-loaded"
          loading="eager"
          onError={[Function]}
          onLoad={[Function]}
          src="//perico.damai.cn/userheadphotos/927551/185510204.jpg"
          style={
            Object {
              "objectFit": "cover",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`Badge snapshot Badge demo snapshot 0 6`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "row",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "20",
    }
  }
>
  <div
    className="bui-badge bui-badge-position"
  >
    <div
      className="bui-badge-primary bui-badge-outlined bui-badge-standard"
    >
      12
    </div>
    <div
      className="bui-avatar bui-avatar-medium bui-avatar-rounded"
    >
      <div
        className="bui-image bui-avatar-img"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      >
        <img
          className="bui-image-img bui-image-loaded"
          loading="eager"
          onError={[Function]}
          onLoad={[Function]}
          src="//perico.damai.cn/userheadphotos/927551/185510204.jpg"
          style={
            Object {
              "objectFit": "cover",
            }
          }
        />
      </div>
    </div>
  </div>
  <div
    className="bui-badge bui-badge-position"
  >
    <div
      className="bui-badge-primary bui-badge-outlined bui-badge-bubble"
    >
      12
    </div>
    <div
      className="bui-avatar bui-avatar-medium bui-avatar-rounded"
    >
      <div
        className="bui-image bui-avatar-img"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      >
        <img
          className="bui-image-img bui-image-loaded"
          loading="eager"
          onError={[Function]}
          onLoad={[Function]}
          src="//perico.damai.cn/userheadphotos/927551/185510204.jpg"
          style={
            Object {
              "objectFit": "cover",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`Badge snapshot Badge demo snapshot 0 7`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
    }
  }
>
  <div
    className="bui-badge bui-badge-position"
  >
    <div
      className="bui-badge-primary bui-badge-contained bui-badge-standard"
    >
      99+
    </div>
    <div
      className="bui-avatar bui-avatar-medium bui-avatar-rounded"
    >
      <div
        className="bui-image bui-avatar-img"
        style={
          Object {
            "height": undefined,
            "width": undefined,
          }
        }
      >
        <img
          className="bui-image-img bui-image-loaded"
          loading="eager"
          onError={[Function]}
          onLoad={[Function]}
          src="//perico.damai.cn/userheadphotos/927551/185510204.jpg"
          style={
            Object {
              "objectFit": "cover",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`Badge snapshot Badge demo snapshot 0 8`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "row",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "10px",
    }
  }
>
  <div
    className="bui-badge bui-badge-position"
  >
    <div
      className="bui-badge-success bui-badge-contained bui-badge-standard"
    >
      5
    </div>
    <svg
      className="bui-svg-icon"
      color="#00d68f"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
        }
      }
      focusable="false"
      style={
        Object {
          "fontSize": "38px",
        }
      }
      viewBox="0 0 96 96"
    />
  </div>
  <div
    className="bui-badge bui-badge-position"
  >
    <div
      className="bui-badge-warning bui-badge-contained bui-badge-standard"
    >
      5
    </div>
    <svg
      className="bui-svg-icon"
      color="#ff8533"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
        }
      }
      focusable="false"
      style={
        Object {
          "fontSize": "38px",
        }
      }
      viewBox="0 0 96 96"
    />
  </div>
  <div
    className="bui-badge bui-badge-position"
  >
    <div
      className="bui-badge-info bui-badge-contained bui-badge-standard"
    >
      5
    </div>
    <svg
      className="bui-svg-icon"
      color="#148aff"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
        }
      }
      focusable="false"
      style={
        Object {
          "fontSize": "38px",
        }
      }
      viewBox="0 0 96 96"
    />
  </div>
  <div
    className="bui-badge bui-badge-position"
  >
    <div
      className="bui-badge-primary bui-badge-contained bui-badge-standard"
    >
      5
    </div>
    <svg
      className="bui-svg-icon"
      color="#ff335c"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
        }
      }
      focusable="false"
      style={
        Object {
          "fontSize": "38px",
        }
      }
      viewBox="0 0 96 96"
    />
  </div>
  <div
    className="bui-badge bui-badge-position"
  >
    <div
      className="bui-badge-danger bui-badge-contained bui-badge-standard"
    >
      5
    </div>
    <svg
      className="bui-svg-icon"
      color="#ff335c"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
        }
      }
      focusable="false"
      style={
        Object {
          "fontSize": "38px",
        }
      }
      viewBox="0 0 96 96"
    />
  </div>
</div>
`;

exports[`Badge snapshot Badge demo snapshot 0 9`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "row",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "10px",
    }
  }
>
  <div
    className="bui-badge bui-badge-position"
  >
    <div
      className="bui-badge-success bui-badge-outlined bui-badge-standard"
    >
      5
    </div>
    <svg
      className="bui-svg-icon"
      color="#00d68f"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
        }
      }
      focusable="false"
      style={
        Object {
          "fontSize": "38px",
        }
      }
      viewBox="0 0 96 96"
    />
  </div>
  <div
    className="bui-badge bui-badge-position"
  >
    <div
      className="bui-badge-warning bui-badge-outlined bui-badge-standard"
    >
      5
    </div>
    <svg
      className="bui-svg-icon"
      color="#ff8533"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
        }
      }
      focusable="false"
      style={
        Object {
          "fontSize": "38px",
        }
      }
      viewBox="0 0 96 96"
    />
  </div>
  <div
    className="bui-badge bui-badge-position"
  >
    <div
      className="bui-badge-info bui-badge-outlined bui-badge-standard"
    >
      5
    </div>
    <svg
      className="bui-svg-icon"
      color="#148aff"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
        }
      }
      focusable="false"
      style={
        Object {
          "fontSize": "38px",
        }
      }
      viewBox="0 0 96 96"
    />
  </div>
  <div
    className="bui-badge bui-badge-position"
  >
    <div
      className="bui-badge-primary bui-badge-outlined bui-badge-standard"
    >
      5
    </div>
    <svg
      className="bui-svg-icon"
      color="#ff335c"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
        }
      }
      focusable="false"
      style={
        Object {
          "fontSize": "38px",
        }
      }
      viewBox="0 0 96 96"
    />
  </div>
  <div
    className="bui-badge bui-badge-position"
  >
    <div
      className="bui-badge-danger bui-badge-outlined bui-badge-standard"
    >
      5
    </div>
    <svg
      className="bui-svg-icon"
      color="#ff335c"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
        }
      }
      focusable="false"
      style={
        Object {
          "fontSize": "38px",
        }
      }
      viewBox="0 0 96 96"
    />
  </div>
</div>
`;
